tasks/deploy/config/mainnet/morpho-blue-wsteth-usdc-182day.ts

```diff
@@ -0,0 +1,126 @@
+import { Address, encodeAbiParameters, parseEther, zeroAddress } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    MAINNET_USDC_ADDRESS,
+    MAINNET_WSTETH_ADDRESS,
+    SIX_MONTHS,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import { MAINNET_FACTORY_NAME } from "./factory";
+import { MAINNET_MORPHO_BLUE_COORDINATOR_NAME } from "./morpho-blue-coordinator";
+
+export const MAINNET_MORPHO_BLUE_WSTETH_USDC_182DAY_NAME =
+    "ElementDAO 182 Day Morpho wstETH/USDC Hyperdrive";
+
+const CONTRIBUTION = parseEther("100");
+
+const morphoBlueParameters = encodeAbiParameters(
+    [
+        {
+            components: [
+                {
+                    name: "morpho",
+                    type: "address",
+                },
+                {
+                    name: "collateralToken",
+                    type: "address",
+                },
+                {
+                    name: "oracle",
+                    type: "address",
+                },
+                {
+                    name: "irm",
+                    type: "address",
+                },
+                {
+                    name: "lltv",
+                    type: "uint256",
+                },
+            ],
+            name: "MorphoBlueParams",
+            type: "tuple",
+        },
+    ],
+    [
+        {
+            morpho: "0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb" as `0x${string}`,
+            collateralToken: MAINNET_WSTETH_ADDRESS,
+            oracle: "0x48F7E36EB6B826B2dF4B2E630B62Cd25e89E40e2" as `0x${string}`,
+            irm: "0x870aC11D48B15DB9a138Cf899d20F13F79Ba00BC" as `0x${string}`,
+            lltv: BigInt("860000000000000000"),
+        },
+    ],
+);
+
+export const MAINNET_MORPHO_BLUE_WSTETH_USDC_182DAY: HyperdriveInstanceConfig<"MorphoBlue"> =
+    {
+        name: MAINNET_MORPHO_BLUE_WSTETH_USDC_182DAY_NAME,
+        prefix: "MorphoBlue",
+        coordinatorAddress: async (hre) =>
+            hre.hyperdriveDeploy.deployments.byName(
+                MAINNET_MORPHO_BLUE_COORDINATOR_NAME,
+            ).address,
+        deploymentId: toBytes32(
+            MAINNET_MORPHO_BLUE_WSTETH_USDC_182DAY_NAME.slice(0, 32),
+        ),
+        salt: toBytes32("0x42080085"),
+        extraData: morphoBlueParameters,
+        contribution: CONTRIBUTION,
+        fixedAPR: parseEther("0.03"),
+        timestretchAPR: parseEther("0.035"),
```

Comment:
Let's bump this to 7.5%. This will smoothly support rates up to 30% (this is the high end of the supply rate all time). It can go higher with less liquidity.